packages/math-input/src/components/keypad/shared-keys.tsx

```diff
@@ -37,9 +39,19 @@ function getCursorContextConfig(cursorContext?: CursorContext) {
 }
 
 export default function SharedKeys(props: Props) {
-    const {onClickKey, cursorContext, divisionKey, multiplicationDot} = props;
+    const {
+        onClickKey,
+        cursorContext,
+        divisionKey,
+        multiplicationDot,
+        selectedPage,
+    } = props;
 
     const cursorKeyConfig = getCursorContextConfig(cursorContext);
+    const fracCoord: readonly [number, number] =
```

Comment:
It might be good to add a comment here to explain fracCoord for our future selves. :) Not a blocker at all.